tests/4.5/target_teams_distribute/test_target_teams_distribute_depend_disjoint_section.F90

```diff
@@ -0,0 +1,85 @@
+!===--- test_target_teams_distribute_depend_disjoint_section.F90------------===//
+!
+! OpenMP API Version 4.5 Nov 2015
+!
+! This test checks the out-out dependency of two tasks when the array
+! sections in the depend lists are disjoint (non-overlapping). If no
+! asynchronous behavior can be shown, then the test gives only a warning,
+! since this is still complaint. This test will always pass.
```

Comment:
It seems like this test can fail if the value is neither one or the other.

Reply:
Ah yes, I forgot to change the description after adding that error condition.

Reply:
If you want it to fail you need OMPVV_TEST(_AND_SET), OMPVV_ERROR does not guarantee that the test will fail, it just displays an error.

Reply:
Should be fixed now